content/faq/how-do-I-use-composer-with-github-access-token.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How do I use Composer with GitHub personal access token?
+menu:
+  main:
+    parent: 'faq'
+    weight: 40
+weight: 40
+linkTitle: Setup GitHub token
+---
+If you need to download private repositories from GitHub during the build
+process, an access token has to be available during `composer install`.
+
+To set up that, make sure you are using one of the Docker images from
+[ContinuousPipe dockerfiles](https://github.com/continuouspipe/dockerfiles)
+as base image. These images are prepared to take a GITHUB_TOKEN 
+build argument which is used to set the token like this:
+`composer global config github-oauth.github.com "$GITHUB_TOKEN"`
+
+Note: Docker build arguments are used only during the build process 
+and will not be included in the built image. However the composer 
+call above will save it in the filesystem of the image. So the final
+Docker images should be pushed into a private repository.
+
+Let's suppose you have a service called `api` which requires an access 
+token.
+Open the flow configuration and add a new variable named `GITHUB_TOKEN`.
+After that update the Yaml file to pass it as an environment variable:
+``` yaml
+# continuous-pipe.yml
+tasks:
+    images:
+        build:
+            services:
+                api:
+                    environment:
+                        - name: GITHUB_TOKEN
+                          value: ${GITHUB_TOKEN}
+
+            # ...
+```
+
+When you build the image locally with Docker Compose, then create an environment
```

Comment:
possibly link to https://docs.docker.com/compose/ here as I don't think this is mentioned elsewhere in the docs